lib/data_exporter.rb

```diff
@@ -76,10 +76,25 @@ def travel_advice_csv(subscriber_lists, at: nil)
 
   def present_travel_advice_report(subscriber_list, at: nil)
     {
-      "title" => cleanup(subscriber_list.title),
+      title:        cleanup(subscriber_list.title),
```

Comment:
Code formatting question: do the values need to all be in line with each other?

Reply:
they don't have to, no, it's just something I like doing, but also happy not to :)

Reply:
Not sure if we have a preferred option on GOVUK - lets go with what's used most commonly elsewhere 🙂